README.md

```diff
@@ -74,15 +74,27 @@ For the purpose of setup, your project's root folder (where you can find your `.
         ...
     end
     ```
-4. Then run the following command to installl pods:
+4. (Optional) If you want to use background blur and replacement features, add:
+    ```
+    target 'YourTarget' do
+      pod 'AmazonChimeSDKMachineLearning-Bitcode'
```

Comment:
Is there ML framework without Bitcode?

Reply:
Oh good point there is. Let me add that instruction

Reply:
I think the SDK is also written for bitcode, it might make sense to put the same. There is also AmazonChimeSDKMachineLearning without bitcode.